atomist/try-atomist.md

```diff
@@ -0,0 +1,104 @@
+> purposes. For further evaluation or production use, integrate Atomist with
+> your container registry. See [get started](./get-started.md).
+
+1. Go to the [Atomist website](https://dso.docker.com) and sign in using your
+   Docker ID.
+2. Open the **Integrations** tab.
+3. Under **API Keys**, create a new API key.
+4. In your terminal of choice, invoke the Atomist CLI tool using `docker run`.
+   Update the following values:
+
+   - `--workspace`: the workspace ID found on the **Integrations** page on the
+     Atomist website.
+   - `--api-key`: the API key you just created.
+   - `--image`: the Docker image that you want to index.
+
+   ```bash
+   docker run \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      -ti atomist/docker-registry-broker:0.0.1 \
+      index-image local \
+      --workspace AQ1K5FIKA \
+      --api-key team::6016307E4DF885EAE0579AACC71D3507BB38E1855903850CF5D0D91C5C8C6DC0 \
+      --image docker.io/david/myimage:latest
+   ```
+
+   > Note
+   >
+   > The image must have a tag (for example, `myimage:latest`) so that you are
+   > able to identify the image in the
+   > [Atomist web UI](https://dso.docker.com/r/auth/overview/images).
+
+   The output should be similar to the following:
+
+   ```bash
+   [info] Starting session with correlation-id c12e08d3-3bcc-4475-ab21-7114da599eaf
+   [info] Starting atomist/docker-vulnerability-scanner-skill 'index_image' (1f99caa) atomist/skill:0.12.0-main.44 (fe90e3c) nodejs:16.15.0
+   [info] Indexing image python:latest
+   [info] Downloading image
+   [info] Download completed
+   [info] Indexing completed
+   [info] Mapped packages to layers
+   [info] Indexing completed successfully
+   [info] Transacting image manifest for docker.io/david/myimage:latest with digest sha256:a8077d2b2ff4feb1588d941f00dd26560fe3a919c16a96305ce05f7b90f388f6
+   [info] Successfully transacted entities in team AQ1K5FIKA
+   [info] Image URL is https://dso.atomist.com/AQ1K5FIKA/overview/images/myimage/digests/sha256:a8077d2b2ff4feb1588d941f00dd26560fe3a919c16a96305ce05f7b90f388f6
+   [info] Transacting SBOM...
+   [info] Successfully transacted entities in team AQ1K5FIKA
+   [info] Transacting SBOM...
+   ```
+
+5. When the command exits, open the
+   [Atomist web UI](https://dso.docker.com/r/auth/overview/images), where you
+   should see the image in the list.
+
+   ![indexed image in the image overview list](./images/images-overview.png){:
+   width="700px"}
+
+6. Select the image name. This gets you to the list of image tags.
+
+   ![list of image tags](./images/tags-list.png){: width="700px"}
+
+   Since this is your first time indexing this image, the list only has one tag
+   for now. When you integrate Atomist with your container registry, images and
+   tags show up in this list automatically.
+
+7. Select the tag name. This shows you the insights for this tag.
+
+   ![vulnerability breakdown view](./images/vulnerabilities-overview.png){:
```

Comment:
aaahh, totally right! We should probably have an instruction that describes the different views, especially the breakdown page... though I think it can wait a bit since the DSO web UI is not the main priority at the moment and the UI will likely undergo some changes